server/publications.coffee

```diff
@@ -37,6 +37,7 @@ Meteor.publish 'articleIncidents', (articleId) ->
   Incidents.find
     articleId: articleId
     deleted: $in: [null, false]
+    accepted: true
```

Comment:
The unaccepted incidents are used on the curator inbox to underline potential counts.